src/main/java/ee/tuleva/onboarding/mandate/application/ApplicationService.java

```diff
@@ -104,6 +105,18 @@ public boolean hasPendingWithdrawals(Person person) {
         .collect(toList());
   }
 
+  private FundDto getTargetFund(MandateFundsTransferExchangeDTO exchangeDTO, String language) {
+    String targetFundIsin = exchangeDTO.getTargetFundIsin();
+    if (targetFundIsin == null) return null;
+
+    Fund targetFund = fundRepository.findByIsin(targetFundIsin);
+    if (targetFund == null) {
+      throw new IllegalArgumentException(
+          "Fund with ISIN \"" + targetFundIsin + "\" not found in database!");
```

Comment:
I like to keep the variables at the end of the log message to keep them easily _greppable_. 

Right now if you would like to find all instances of this exception from Papertrail, you would have to search for `"Fund with ISIN"` -> and it might find other logging statements instead as well - since it's a fairly generic phrase. 
Or search for `"not found in database"` which is also pretty generic and might find other log lines as well.

However, if you would log it like this:
```
throw new IllegalArgumentException("Fund not found in the database: targetFundIsin=" + targetFundIsin);
```
It would be very easy to search for this exact error message from Papertrail by the _specific_ search string:
"Fund not found in the database: targetFundIsin="

I hope I'm making sense with this. The generic logging pattern would be: 
```Error message: var1={var1}, var2={var2}...```